backend/tests/Services/IMDBTest.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+  use App\Services\IMDB;
+  use Illuminate\Foundation\Testing\DatabaseMigrations;
+
+  class IMDBTest extends TestCase {
+
+    use DatabaseMigrations;
+
+    /** @test */
+    public function it_should_parse_imdb_rating()
```

Comment:
Add two more tests. One where imdb has no entry, and one where the rating is e.g 7,0 and you expect 7,0 as a result (and not just 7)